### PR TITLE
Support package installation in multiple buckets

### PR DESCRIPTION
**Pull request by Aminelouati**

### CODE REVIEW

File: rest/rest-cli/src/main/java/org/ow2/proactive_grid_cloud_portal/cli/cmd/sched/InstallPackageCommand.java [hunk 1]
```diff
@@ -76,6 +76,8 @@ public void execute(ApplicationContext currentContext) throws CLIException {
 
             schedulerProperties = retrieveSchedulerProperties(currentContext, scheduler);
 
+            schedulerProperties = addSessionIdToSchedulerProperties(currentContext, schedulerProperties);
```
Comment (marcocast) [inline rest/rest-cli/src/main/java/org/ow2/proactive_grid_cloud_portal/cli/cmd/sched/InstallPackageCommand.java]: no need to assign schedulerProperties = ...  if you pass schedulerPropertiesas parameters. It's a p[ass by reference argument

File: rest/rest-cli/src/main/java/org/ow2/proactive_grid_cloud_portal/cli/cmd/sched/InstallPackageCommand.java [hunk 2]
```diff
@@ -149,4 +151,10 @@ private ScriptResult executeScript(Map<String, Object> schedulerProperties) thro
             SchedulerRestInterface scheduler) throws PermissionRestException, NotConnectedRestException {
         return scheduler.getSchedulerPropertiesFromSessionId(currentContext.getSessionId());
     }
+
+    private Map<String, Object> addSessionIdToSchedulerProperties(ApplicationContext currentContext,
```
Comment (marcocast) [inline rest/rest-cli/src/main/java/org/ow2/proactive_grid_cloud_portal/cli/cmd/sched/InstallPackageCommand.java]: this should be void

File: tools/LoadPackage.groovy [hunk 3]
```diff
@@ -22,8 +22,9 @@ class LoadPackage {
     private final File WORKFLOW_TEMPLATES_DIR
     private final String WORKFLOW_TEMPLATES_DIR_PATH
     private final String CATALOG_URL
+    private final String SESSION_ID
 
-
+    private String sessionId
```
Comment (marcocast) [inline tools/LoadPackage.groovy]: There is already a variable called SESSION_ID, what is the difference?

File: tools/LoadPackage.groovy [hunk 4]
```diff
@@ -141,143 +147,147 @@ class LoadPackage {
...
+            def bucket_found = catalog_buckets_list.find { object -> object.name == bucket }
+            writeToOutput(" bucket_list " + bucket + " found? " + (bucket_found != null))
+
+            // Create a bucket_list if needed -------------
+            def bucket_id = null
+            if (bucket_found) {
+                bucket_id = bucket_found.id
+            } else {
                 // POST QUERY
-                def query_push_obj_query = this.CATALOG_URL + "/buckets/" + bucket_id + "/resources?name=" + object_name + "&kind=" + kind + "&commitMessage=" + commitMessageEncoded + "&contentType=" + contentType
```
Comment (marcocast) [inline tools/LoadPackage.groovy]: this method is too big and not readable, needs some refactor :)

**Comment by Aminelouati** [comment]
This PR was divided in smaller PRs

**Pull request closed without merging**